lib/masamune/tasks/elastic_mapreduce_thor.rb

```diff
@@ -36,7 +36,10 @@ class ElasticMapreduceThor < Thor
     class_option :template, :type => :string, :aliases => '-t', :desc => 'Execute named template command'
     class_option :params, :type => :hash, :aliases => '-p', :desc => 'Bind params to named template command', :default => {}
     def elastic_mapreduce_exec
-      elastic_mapreduce(options.merge(interactive: true, extra: extra_or_ssh))
+      elastic_mapreduce_options = options.dup.with_indifferent_access
```

Comment:
Thor freezes the `option`s hash before this point- dup is to avoid illegal modification exceptions